formbricks/formbricks
# feat: Make.com integration

## What does this PR do?

Fixes #599 

1. Adds make to formbricks Integration page
2. Adds a new section for make.com in documentation
3. Created an app for formbricks on make.com (under review) - [Apps public link](https://eu2.make.com/app/invite/67538c04a8ed934e86d4ba8cdd8f8732)


![Screenshot from 2023-08-01 12-03-49](https://github.com/formbricks/formbricks/assets/67850763/972841d2-a091-48b8-be47-2372e09b3aad)

![make](https://github.com/formbricks/formbricks/assets/67850763/3d944d0a-e86e-4775-9df3-e6787b20bf78)




## Type of change




## How should this be tested?



Check Integration page and Integration docs
[Check app on make.com](https://eu2.make.com/app/invite/67538c04a8ed934e86d4ba8cdd8f8732)

## Checklist
